components/form/Label.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export function FloatLabel({ errorLabel, htmlFor, label }) {
```

Comment:
For labels, we use a single component called Label (we don't need the FloatLabel and CheckboxLabel).

The Label component should have `name`, `htmlFor`, `text` (not label), `className` and `floatingLabel` as props.

Take note of the following points when you are updating the file:

- From the `name` props, we can confirm if the `name` label has an error from `FormContext`, so that solves the issue of `errorLabel`.

- If `htmlFor` is not given, we should use the value of `name`. This reduces the chance of repeating the same value over and over again.

- using `text` instead of `label` makes the label text more explanatory in the future

- if `floatingLabel` is set to true then we should use the class defined for floatingLabel, else we should use the class properties for a normal Label. This makes it easier to manage the conversion from floating label to normal labels.